static/app/components/replays/replayController.tsx

```diff
@@ -60,27 +61,28 @@ function ReplayCurrentTime() {
 
 function ReplayPlaybackSpeed({speedOptions}: {speedOptions: number[]}) {
   const {setSpeed, speed} = useReplayContext();
-
```

Comment:
nit: the line break before return is nice. But we don't enforce anything here so 🤷